setup.cfg

```diff
@@ -63,7 +63,12 @@ include = torchgeo*
 # Optional dataset requirements
 datasets =
     h5py
+    # loading .las point clouds (idtrees)
+    laspy>=2.0.0
+    # open3d will add add support for python 3.9 in pypi in v0.14
+    open3d;python_version<'3.9'
     opencv-python
+    pandas>=1.0
```

Comment:
Because it was the oldest version I could install successfully on Python 3.9 and run the tests. Do you want me to add that as a comment?

Reply:
See what the oldest version is for Python 3.6. If all versions that you're able to test work, we don't need a constraint since the constraint is unknown.

Reply:
v0.25.3 works on Python 3.6 🤷

Reply:
It may end up being the first version that adds Python 3 support, that's what I did for shapely